src/components/tree/phyloTree/renderers.js

```diff
@@ -86,37 +86,45 @@ export const drawVaccines = function drawVaccines() {
     .on("click", this.callbacks.onTipClick);
 };
 
+const BATCH_SIZE = 100;
```

Comment:
I think this line should be moved to globals.js

Reply:
Done! :D